lib/__tests__/_sendEvent.node.test.ts

```diff
@@ -1,23 +1,23 @@
 /**
  * @jest-environment node
  */
-import { getInstance } from "../_instance";
+import { getInstanceForNode as getInstance } from "../_instance";
 
 const credentials = {
   apiKey: "testKey",
   appId: "testId"
 };
 
 describe("_sendEvent in node env", () => {
-  let AlgoliaInsights;
+  let aa;
   beforeEach(() => {
-    AlgoliaInsights = getInstance();
-    AlgoliaInsights.init(credentials);
+    aa = getInstance();
+    aa("init", credentials);
   });
 
   it("throws when user token is not set", () => {
     expect(() => {
-      (AlgoliaInsights as any).sendEvent("click", {
+      aa("sendEvent", "click", {
```

Comment:
I see you have migrated this one to the exposed API but not the other ones. Is there reason for not having all tests using the same API ?